### PR TITLE
Import sources from python-pipeline-utils, add gradle build, Jenkinsfile

### PR DESCRIPTION
**Pull request by nre-ableton**
ptal @AbletonDevTools/gotham-city, thanks!

### CODE REVIEW

**Comment by nre-ableton** [comment]
re-ping @AbletonDevTools/gotham-city

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,42 @@
+@Library(['ableton-utils@0.1.0', 'codenarc@0.5.0']) _
```
Comment (ala-ableton) [inline Jenkinsfile]: Let's use `groovylint`.

Comment (nre-ableton) [reply Jenkinsfile]: Ah crap, I didn't even notice that. Thanks!

**Comment by nre-ableton** [comment]
@ala-ableton Fixed!

**Comment by ala-ableton** [comment]
LGTM :+1:

**Pull request merged**